### PR TITLE
[TASK] Add RuboCop checking to the CI

### LINKED ISSUE #14: Use Rubocop

### PR DESCRIPTION
**Pull request by oliverklee**
This commit only add the checks. It does not fix the warnings yet,
but whitelists them instead so they can be fixed in separate PRs.

Fixes #14

### CODE REVIEW

File: page_title_helper.gemspec [hunk 1]
```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'appraisal'
   s.add_development_dependency 'rake', '>= 10.3.2'
   s.add_development_dependency 'shoulda'
+  s.add_development_dependency 'rubocop', '>= 0.47.1'
```
Comment (lwe) [inline page_title_helper.gemspec]: as we don't have a Gemfile.lock, wouldn't it make more sense to use '~> 0.47' or ~> 0.47.1, or we intentionally always want the latest version?

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ gemfile:
 
 script:
   - bundle exec rake --trace test
+  - bundle exec rubocop gemfiles/*.gemfile lib/ test/ Appraisals Gemfile init.rb Rakefile
```
Comment (lwe) [inline .travis.yml]: ^^ remind me to delete the `init.rb` file ;)

Comment (oliverklee) [inline .travis.yml]: Done (in #57). :-)

File: page_title_helper.gemspec (hunk 1 above)
Comment (oliverklee) [inline page_title_helper.gemspec]: Good point. Done.

**Comment by oliverklee** [comment]
Fixed and rebased.

**Comment by lwe** [comment]
👍 nice, thanks

**Pull request merged**